.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+*NOTE: This pull request template is being created and may not be complete.*
+
+## Author checklist (to be completed by original Author)
+- [ ] I have signed the Rocky Open Source Contributor Agreement (ROSCA).
```

Comment:
Should we link this to https://accounts.rockylinux.org/ ?

Reply:
Possible yes though I don't want to block PRs for that unnecessarily. I don't think `xsos` output is subject to any licensing issues as an example. We need to think about how to delineate PRs requiring ROSCA and those which do not.

Reply:
I have an ask to legals about turning down the volume of the CLA a bit with regards to sublicensing.

Reply:
@gmkurtzer Would you prefer we hold off on merge until this is decided? It's a checklist and doesn't (currently) block via any automation our ability to approve the PR.

Reply:
Group meeting decided to open an issue and track this as a separate conversation until we know the best solution.